### PR TITLE
Ideas template for LiberTEM

### PR DESCRIPTION
**Pull request by uellue**
Hi GSoC team, LiberTEM would like to participate again this year. :-)

See also https://github.com/LiberTEM/LiberTEM/issues/543

CC @sk1p

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -452,6 +452,36 @@ <h2>
...
+                <img class="responsive-img"
+                    src="https://libertem.github.io/LiberTEM/_static/logo.png" />
+                <h2>
+                    LiberTEM
+                </h2>
+                <p>LiberTEM is a Python-based open source platform for high-throughput distributed processing of
+                    large-scale binary data sets such as pixelated scanning transmission electron microscopy (STEM) data.
+                    A responsive web application based on React and Redux provides a GUI, it can be used in Python scripts
+                    and Jupyter notebooks through its API, it uses dask.distributed as its default processing back-end,
+                    and it relies heavily on NumPy for processing and data handling.</p>
```
Comment (sk1p) [inline index.html]: Can we hack this maybe into separate sentences?

How about: "It can be used from Python through its API. The processing back-end uses dask.distributed by default and relies heavily on NumPy and related packages for processing and data handling. A web application based on Tornado, React, and Redux provides a GUI, in addition to using LiberTEM in combination with Jupyter notebooks."

Comment (uellue) [reply index.html]: Nice, much better, thx!

**Comment by warthog9** [comment]
Mostly things I'd love to see get cleaned up on your ideas page:

- It's really long, you might lose some potential students just because they'll see a block of text
- On the project ideas, can you make the Beginner/Intermediate/Advanced stand out more, at my own quick glance it's just hard to parse what's there. As a reference, I think MSS's current PR, and idea's page, did an ok job on that ( https://bitbucket.org/wxmetvis/mss/wiki/GSOC2020/mscolab%20Admin%20Interface%20and%20Chat )
- Maybe use headers to break up the Beginner/Intermediate/Advanced (mostly thinking out loud)

Lets start there, you've got a good list of ideas, and I like that you've been pretty detailed on them, just need to try and get them a bit more parse-able at a glance so it's easier for the students to see what's there

**Comment by warthog9** [comment]
Also, I ended up introducing a conflict in accepting another org, can you resolve that before we push forward?

**Comment by uellue** [comment]
> Also, I ended up introducing a conflict in accepting another org, can you resolve that before we push forward?

This should be resolved with the last commit. :-)

**Comment by uellue** [comment]
> Mostly things I'd love to see get cleaned up on your ideas page:

@warthog9 Thx for the feedback, I'm working on it. :-)

**Comment by uellue** [comment]
@warthog9 The ideas page is updated now. Is it better like that? What else can or should we do with the ideas page or this PR? Thx again for your support. :-)

**Comment by terriko** [comment]
Sorry about the slowness!  I think I thought warthog9 had handled it and he thought I had.  Merged now!

**Comment by uellue** [comment]
@terriko Thank you for merging, no problem! :-)
